docs/tutorials/kinesis.md

```diff
@@ -0,0 +1,184 @@
+#
+version: 0
+
+index_id: gh-archive
+
+doc_mapping:
+  field_mappings:
+    - name: id
+      type: text
+      tokenizer: raw
+    - name: type
+      type: text
+      fast: true
+      tokenizer: raw
+    - name: public
+      type: u64
+      fast: true
+    - name: payload
+      type: json
+      tokenizer: default
+    - name: org
+      type: json
+      tokenizer: default
+    - name: repo
+      type: json
+      tokenizer: default
+    - name: actor
+      type: json
+      tokenizer: default
+    -name: other
+      type: json
+      tokenizer: default
+    - name: created_at
+      type: i64
+      fast: true
+
+indexing_settings:
+  timestamp_field: created_at
+
+search_settings:
+  default_search_fields: []
+```
+
+Execute these Bash commands to download the index config and create the `gh-archive` index.
+
+```bash
+# Download GH Archive index config.
+wget -O gh-archive.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/gh-archive/index-config.yaml
+
+# Create index.
+./quickwit index create --index-confg gh-archive.yaml
+```
+
+## Create Kinesis source
+
+```yaml title="kinesis-source.yaml"
+#
+# Kinesis source config file.
+#
+source_id: kinesis-source
+source_type: kinesis
+params:
+  stream_name: gh-archive
+```
+
+Run these commands to download the source config file and create the source.
+
+```bash
+# Download Kinesis soure config.
+wget https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/gh-archive/kinesis-source.yaml
+
+# Create source.
+./quickwit source create --index gh-archive --source-config kinesis-source.yaml
+```
+
+## Create and populate Kinesis stream
+
+Now, let's create a Kinesis stream and load some events into it.
+
+:::tip
+This step may be fairly slow depending on how much bandwidth is available. You can reduce the volume of data to ingest by downloading a single file from the GH Archive and/or place a `| head -n <number of records> \ ` after the `gunzip` command. You can also speed things up by increasing the number of shards and/or the number of jobs launched by `parallel` (`-j` option).
+:::
+
+```bash
+# Create a stream named `gh-archive` with 3 shards.
+aws kinesis create-stream --stream-name gh-archive-foo --shard-count 8
+
+# Download a few GH Archive files.
+wget https://data.gharchive.org/2022-05-12-{10..12}.json.gz
+
+# Load the events into Kinesis stream
+gunzip -c 2022-05-12*.json.gz | \
+jq -c '.created_at = (.created_at | fromdate) | .public = if .public then 1 else 0 end' | \
+parallel --gnu -j8 -N 500 --pipe \
+'jq --slurp -c \'{"Records": [.[] | {"Data": (. | @base64), "PartitionKey": .id }], "StreamName": "gh-archive"}\' > records-{%}.json && aws kinesis put-records --cli-input-json file://records-{%}.json >> out.log'
+```
+
+## Launch indexing and search services
+
+Finally, execute this command to start Quickwit in server mode.
+
+```bash
+# Launch Quickwit services.
+./quickwit services run
```

Comment:
I got this error
```
Command failed: Failed to deserialize index metadata: `unknown field `region_or_endpoint`, expected one of `stream_name`, `region`, `endpoint`, `shutdown_at_stream_eof`
```

Reply:
Thanks for reporting. Will look into it.